nyx/assetlist.json

```diff
@@ -0,0 +1,28 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_name": "nyx",
+    "assets": [
+      {
+        "description": "NYM Token (NYM) is the Nym Network's native utility token, used as the primary means to incentivize mixnet node operators.",
+        "denom_units": [
+          {
+            "denom": "unym",
+            "exponent": 0
+          },
+          {
+            "denom": "NYM",
+            "exponent": 6
+          }
+        ],
+        "base": "unym",
+        "name": "Nym",
+        "display": "nym",
+        "symbol": "NYM",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/",
```

Comment:
can we please expand this urls to point the image you're uploading?

Reply:
https://raw.githubusercontent.com/cosmos/chain-registry/master/nyx/images/nyx.png

Reply:
updated, apologies for missing that the first time